domains/wiicreation.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Wiicreation is a recreation of the Nintendo Wii made in ElectronJS.",
+  "repo": "https://github.com/C1200/WiicreationWebsite",
+  "owner": {
+    "username": "C1200",
+    "email": "coreyw.ultimatemedia@gmail.com"
+  },
+  "record": {
+    "CNAME": "C1200.github.io/WiicreationWebsite"
```

Comment:
The cname should not be a URL. CNAME in this case should just be `c1200.github.io`

Reply:
Fixed it!